spec/Overview.html

```diff
@@ -876,6 +887,75 @@ <h4>The getRandomValues method</h4>
               </p>
             </div>
           </section>
+          <section id="Crypto-method-randomUUID">
+            <h4>The randomUUID method</h4>
+            <p>
+            The <dfn id="dfn-Crypto-method-randomUUID" data-dfn-for=Crypto>randomUUID</dfn>
+            method generates a new <a data-cite="RFC4122#section-4.4">version 4 UUID</a>
+            and returns its <a data-cite="RFC2141#section-2.2">namespace specific string representation</a>
+            as described in <a data-cite="RFC4122#section-3">section 3</a> of [[RFC4122]].
+            To <dfn data-local-lt="generating a random UUID" data-export=
+            "">generate a random UUID</dfn>:
+            </p>
+            <ol>
+              <li>Let |array| be a [=list=] with 16 elements of the type byte.
```

Comment:
Shouldn't this be a byte sequence as defined in Infra?

Reply:
done.